docusaurus.config.js

```diff
@@ -64,6 +64,11 @@ const config = {
             label: 'Documentation',
           },
           { to: 'https://tech.deriv.com/', label: 'Blog', position: 'left' },
+          {
+            to: '/auth',
+            position: 'left',
```

Comment:
can we change this to `right`? 
it should look better I guess, what do you think?

Reply:
Done